docs/ref/tok.md

```diff
@@ -67,6 +67,20 @@ q)"P"$"2015-10-28D03:55:58.6542"
 2015.10.28D03:55:58.654200000
 ```
 
+## Outside of Domain
```

Comment:
Titles in sentence case pls. This is an initial capital, then all lower case UNLESS the word is a proper noun, i.e. the name of something, e.g. Jo or Microsoft.

```suggestion
## Outside of domain
```